### ISSUE TITLE
Betaflight parameter update follow-up

### ISSUE THREAD

**Issue by kohlerj** [uzh-rpg/rpg_quadrotor_control#81]
@mfaessle, @foehnph  Should we, as we update our flight controller to the newest betaflight version, update the parameter file that we provide to configure betaflight. I would not update it for every version but at least for the one we currently use in our drones. Sounds like a plan(e) ;) ?

**Comment by mfaessle** [comment]
Yes, please do this. I suggest to leave the parameter files that we have in [fc_settings](https://github.com/uzh-rpg/rpg_quadrotor_control/tree/master/bridges/sbus_bridge/fc_settings) but rename them to reflect the betaflight version and then add parameter files for new versions. Please also mention this in the [betaflight wiki page](https://github.com/uzh-rpg/rpg_quadrotor_control/wiki/Betaflight-Firmware). Thanks

**Comment by kohlerj** [comment]
See #82
